vertica_python/tests/test_cursor.py

```diff
@@ -377,6 +378,13 @@ def test_empty_query(self):
             res = cur.fetchall()
             self.assertListOfListsEqual(res, [])
 
+    def test_format_quote_unicode(self):
+        with self._connect() as conn:
+            cur = conn.cursor()
+            bad_word = u'Fr\xfchst\xfcck'
+            formatted_word = u''.join((u'"', re.escape(bad_word), u'"'))
```

Comment:
had to do this because re.escape behavior is not the same between py2/py3: `'Fr\ühstück' != 'Frühstück'`

Reply:
How does `'Fr\ühstück'` come from?

Reply:
side effect of `re.escape()` (note I forgot to add the backslash to the other ü)
![image](https://user-images.githubusercontent.com/2257119/44608978-b6ec0480-a7aa-11e8-8b6e-0f5e8db641e5.png)

Reply:
Ok. I don't think `re.escape()` behavior is different between py2/py3.
`print(re.escape(u'Fr\xfchst\xfcck'))` always return `Fr\ühst\ück`.

Thank you for your contribution. Please rebase your branch so that we can merge, your changes to file `vertica_python/tests/test_cursor.py` should now move to `vertica_python/tests/integration_tests/test_cursor.py `.

Reply:
![image](https://user-images.githubusercontent.com/2257119/44675571-c99e4d80-a9e5-11e8-9180-2905b714f5df.png)

sounds good, will make the test changes

Reply:
Thanks, good to know that `re.escape()` behavior changed in version 3.7. But this project does not officially support python 3.7 currently.